portato-io/portatoapp
# Adds docker container

<h2>Changes proposed in this pull request:</h2>

This PR adds a docker container which launches the empty web app. 
The set-up was built from https://testdriven.io/blog/dockerizing-django-with-postgres-gunicorn-and-nginx/ 

<h2>Performed test by PR author:</h2>

The web app was tested by running 
`
python manage.py migrate
python manage.py runserver
`
and checking that http://localhost:8000/ showed a successful deployment.

To test the containerized version I ran:

`
docker-compose build
docker-compose up -d
`
followed again by checking that http://localhost:8000/ showed a successful deployment.

<h2>Known limitations:</h2>

<h4>Bugs detected in this PR that will be fixed later:</h4>

*Add here + link to timeline event created to fix it*

<h4>Limitations detected in this PR that shall become new expected behavior:</h4>


<h4>Questions to ask yourself when reviewing:</h4>

* Do I understand which problem the merge request will solve?
* Is the work I see complete?
  * No bugs or open corner cases?
  * Is all the needed documentation there?
  * Is the UX/UI according to a mockup by the UX designer or as discussed in the team? Or is it small/simple enough to be decided between myself  and the PR author alone?
  * Are all the user visible messages small and sane or discussed with the UX designer and translatable?
  * Are all dialogs working in the dialogs debug setup?
* Have I tested the code at least once on all relevant use cases, platforms and with mocklink, SITL or an actual plane as necessary or have I double checked with the PR author that this has happened?
* Is the code quality according to coding guidelines?
* Small-but-annoying stuff like typos etc.

<h4>Reactions to review findings:</h4>

* **Opinion Piece**
  * Add [OP] to your comment
  * Code that is OK like it is but could be solved otherwise.
* **Just-do-it**
  * Add [JDI] to your comment
  * Trivial stuff like typos that can be fixed without further review.
* **Regular finding**
  * Add a normal comment
  * A finding that needs changing and discussion. The author is supposed to fix it and let it be reviewed again.
* **Blocker**
  * Add a normal comment and discuss offline asap
  * A finding that the reviewer considers going against good practices or fundamentally flawed. Severe rewriting and fixing are in order
* **Praise**
  * Add an emoji ;)
  * If the code is awesome, give feedback

</details>
